### PR TITLE
Add info about CarPlay critical notifications showing on car's built-in screen

### PR DESCRIPTION
**Pull request by jeremy-lemaire**
Add some info about CarPlay critical notifications showing on screen. I have not seen anything online about this, so I tested it : standard HA notifications aren't shown, while critical notifications played a sound and displayed informations on my car's built-in screen.

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @jeremy-lemaire, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;companion.home-assistant#141).

Once you do that we will be able to review and accept this pull request.

Thanks!

File: docs/notifications/critical.md [hunk 1]
```diff
@@ -30,3 +30,5 @@ automations:
 ```
 
 If you have previously read the [sounds documentation](sounds.md) this syntax should be mostly familiar. Note the example expands the `sound` attribute to include the `critical: 1` flag, and `volume: 1.0` to set the volume to 100 %.
+
+For **CarPlay** users, it's also worth mentioning that critical notifications are the only ones that can appear on the car's built-in display, making them very useful if you want to know when something critical happens while you're driving.
```
Comment (TomBrien) [inline docs/notifications/critical.md]: Just to match styling elsewhere

File: docs/notifications/critical.md [hunk 2]
```diff
@@ -30,3 +30,5 @@ automations:
 ```
 
 If you have previously read the [sounds documentation](sounds.md) this syntax should be mostly familiar. Note the example expands the `sound` attribute to include the `critical: 1` flag, and `volume: 1.0` to set the volume to 100 %.
+
```
Comment (TomBrien) [inline docs/notifications/critical.md]: Just to make this a little more prominent.

```suggestion
### CarPlay
```

**Comment by TomBrien** [comment]
Good spot thank you.

Can you just check this isn't an artefact of Do Not Disturb activated automatically when CarPlay is connected.

**Pull request merged**